### PR TITLE
requests wrapper

### PR DESCRIPTION
**Pull request by hwchase17**

### CODE REVIEW

File: langchain/requests.py [hunk 1]
```diff
@@ -3,71 +3,131 @@
...
 import aiohttp
 import requests
-from pydantic import BaseModel, Extra
+from pydantic import BaseModel
 
 
-class RequestsWrapper(BaseModel):
-    """Lightweight wrapper around requests library."""
-
+class Requests(BaseModel):
```
Comment (hinthornw) [inline langchain/requests.py:9]: Could add a docstring

**Review by hinthornw — APPROVED** [review]
This will be helpful - Thanks! I prefer this to the other PRs in this vein, assuming we get the validation errors to work

File: langchain/requests.py [hunk 2]
```diff
@@ -3,71 +3,131 @@
...
+        return await response.text()
+
+    async def aput(self, url: str, data: Dict[str, Any], **kwargs: Any) -> str:
+        """PUT the URL and return the text asynchronously."""
+        response = await self.requests.aput(url, data, **kwargs)
+        return await response.text()
+
+    async def adelete(self, url: str, **kwargs: Any) -> str:
+        """DELETE the URL and return the text asynchronously."""
+        response = await self.requests.aget(url, **kwargs)
```
Comment (Shrined) [inline langchain/requests.py:128]: Should this be `self.requests.adelete(url, **kwargs)` instead of `self.requests.aget(url, **kwargs)`?

Comment (hwchase17) [reply langchain/requests.py:128]: yes good catch, thanks

**Pull request merged**